src/Services/Auth/index.ts

```diff
@@ -0,0 +1,80 @@
+import { axiosAuthInstance, axiosCommonInstance } from '@/Api/axiosInstance';
+import { DOMAIN, AUTH_TOKEN_KEY } from '@/Constants/Api';
+import { PostLoginRequestType, PostSignUpRequestType } from '@/Types/Request';
+import { UserResponseType } from '@/Types/Response';
+import { UserType } from '@/Types/UserType';
+
+/**
+ * @brief 사용자가 이메일과 비밀번호로 서비스에 로그인합니다.
+ * @return 리턴값은 별도로 존재하지 않습니다.
+ * @todo 빈칸 에러 핸들링을 api함수 내부에서 할지, 외부에서 할지 고민해봐야 합니다.
+ */
+export const login = async ({ email, password }: PostLoginRequestType) => {
+  try {
+    const res = await axiosCommonInstance.post<UserResponseType>(DOMAIN.LOGIN, {
+      email,
+      password,
+    });
+
+    const { token } = res.data;
+    sessionStorage.setItem(AUTH_TOKEN_KEY, token);
+  } catch (e) {
+    console.error(e);
+  }
+};
```

Comment:
오... 로그인 API를 엄청 편하게 작성해주셨네요!!

근데 로그인 API 함수에서 통신, 세션 스토리지 관리 까지 하는 것 보다 그냥 res 데이터를 반환해주고 사용하는 곳에서 세션 작업을 하는게 어떨까요? 

그리고 아마 로그인 시 전역 상태 관리에도 등록해야해서 유저 데이터를 반환하는 것이 더 좋아 보입니다!

Reply:
네 저도 지금은 로그인 테스트도 할 겸 해서 로그인 API 함수 내부에서 세션스토리지 작업까지 넣었는데, 이 부분은 분리해서 인혁님이 만드시는 로그인 컴포넌트 쪽에서 해주시는게 좋을 것 같습니다! 위에서 말씀 해주신 것처럼 API 함수 내부에서는 최대한 통신 로직만 처리하는게 유지보수에 용이할 것 같습니다.

해당 부분은 리뷰 반영해서 이렇게 수정하도록 하겠습니당.
```suggestion
export const login = async ({ email, password }: PostLoginRequestType) => {
  try {
    const res = await axiosCommonInstance.post<UserResponseType>(DOMAIN.LOGIN, {
      email,
      password,
    });

    return res.data;
  } catch (e) {
    console.error(e);
    return null;
  }
};
```